samples/highcharts/chartchooser/radial-bar-monochrome/demo.css

```diff
@@ -0,0 +1,50 @@
+#container {
```

Comment:
Now `export-data` is working, but it is not working well because
a) We removed the styles (Click burger menu => View data table)
b) There is HTML in the categories. Best practice is to leave the categories clean, and add presentational stuff in formatters/formats. Unfortunately in this case I can't see a way to use `label.format`, so if we choose to keep the `export-data` module, we need to use `label.formatter` to add the flags.